MaterialLib/MPL/Properties/SaturationDependentSwelling.cpp

```diff
@@ -0,0 +1,157 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "MaterialLib/MPL/Properties/SaturationDependentSwelling.h"
+
+#include <algorithm>
+#include <iostream>
```

Comment:
Is this include necessary?

Reply:
oh, no. My favorite ;)
Dropped also the algorithm...